app/views/chargings/_chargings_table.html.slim

```diff
@@ -0,0 +1,19 @@
+            p[class="h4"] Chargings
+            table[class="table table-striped table-dark"]
+              thead
+                tr
+                  th[scope="col"] Station 
+                  th[scope="col"] Amount
+                  th[scope="col"] Date
+                  th[scope="col"] Actions
+                tbody
+                - for item in @chargings
+                  tr
+                    td.brand_station = item.brand_station
+                    td.amount
+                      | $
+                      = item.amount
+                    td.date = item.date
+                    td.actions
+                      a[href="#" class="editAction" id="#{item.id}"] Edit 
```

Comment:
Nothing happens when I click the link

Reply:
Could you please share the errors from your browser's console if there are any, when you click 'Edit'

Reply:
And I think it happens because id field is null
<img width="420" alt="Screenshot 2022-09-05 at 17 25 46" src="https://user-images.githubusercontent.com/88270887/188481304-f3617c40-279e-4571-9697-49b8048aa6a7.png">

Reply:
It works for me on Firefox and Google Chrome. Are you using Safari ?

Reply:
I was using Chrome, Safari shows it slightly differently, but the error is the same
![Uploading Screenshot 2022-09-06 at 11.09.49.png…]()

Reply:
<img width="512" alt="Screenshot 2022-09-06 at 11 09 49" src="https://user-images.githubusercontent.com/88270887/188595980-d71b0ee3-e343-4068-8dee-7fa3a0fbddae.png">

Reply:
Can you please check if the Edit link has and id="..". And Chrome version, I will test it with your version

Reply:
Nevermind, I forgot that I commented out the hidden filed 🙄